sbncode/FlashMatch/template_generators/generate_simple_weighted_template.py

```diff
@@ -632,35 +656,64 @@ def main():
     global xbin_width
     global time_delay
     global tolerable_time_diff
+    global suffix_list
+    global ftype_long
     if args.sbnd:
-        fcl_params = fhicl.make_pset('flashmatch_sbnd.fcl')
-        pset = dotDict(fcl_params['sbnd_simple_flashmatch'])
         detector = "sbnd"
-        time_delay = 0.15
-        dir = rootfile.Get(file_updated+":/fmatch")
-        nuslice_tree = dir.Get("nuslicetree")
+        metrics_filename = 'fm_metrics_' + detector + '.root'
+        hfile_top = TFile(metrics_filename, 'RECREATE',
+                      'Simple flash matching metrics for ' + detector.upper())
+        hfile_top.Close()
+        suffix_list = ["", "_op", "_ara", "_opara"]
+        time_delays = [0.15, 0, 0.04, 0]
```

Comment:
This hardcoding for SBND should have a TODO note so that it can be amended in the future.